app/components/SliderQuestions.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import RangeSlider from '~/components/RangeSlider';
+
+interface SliderQuestionsProps {
+    symptom: string;
+    questions: {
+        text: string;
+        answers: string[];
+    }[];
+    onSliderChange: (value: number) => void;
+}
+
+const SliderQuestions: React.FC<SliderQuestionsProps> = ({ symptom, questions, onSliderChange }) => {
+    return (
+        <div className="w-[45vw] m-10">
+            {questions.map((question, index) => (
+                <div key={index}>
```

Comment:
Instead of all SliderQuestions as a component and then just the range slider under them, make each question a component